lib/bindings/src/breez_sdk_liquid.udl

```diff
@@ -438,11 +439,13 @@ interface SendDestination {
 
 dictionary PrepareSendResponse {
     SendDestination destination;
-    u64 fees_sat;
+    u64? fees_sat;
```

Comment:
Why optional?

Reply:
There might be insufficient bitcoin funds to estimate paying with satoshi